### ISSUE TITLE
ETA Expanding Data methods causes T14682 to fail

### ISSUE THREAD

**Issue by mpickering** [tweag/ghc#42]
It is just a stderr mismatch error but no point accepting it now.

```
Actual stderr output differs from expected:
diff -uw "./deriving/should_compile/T14682.run/T14682.stderr.normalised" "./deriving/should_compile/T14682.run/T14682.comp.stderr.normalised"
--- ./deriving/should_compile/T14682.run/T14682.stderr.normalised	2018-04-27 09:48:05.341872643 +0000
+++ ./deriving/should_compile/T14682.run/T14682.comp.stderr.normalised	2018-04-27 09:48:05.341872643 +0000
@@ -22,8 +22,8 @@
   
   instance Data.Data.Data T14682.Foo where
     Data.Data.gfoldl k z (T14682.Foo a1 a2)
-      = ((z T14682.Foo `k` a1) `k` a2)
-    Data.Data.gunfold k z _ = k (k (z T14682.Foo))
+      = ((z (/ a1 a2 -> T14682.Foo a1 a2) `k` a1) `k` a2)
+    Data.Data.gunfold k z _ = k (k (z (/ a1 a2 -> T14682.Foo a1 a2)))
     Data.Data.toConstr (T14682.Foo _ _) = T14682.$cFoo
     Data.Data.dataTypeOf _ = T14682.$tFoo
   
*** unexpected failure for T14682(normal)
```

**Comment by aspiwack** [comment]
Probably already fixed.

**Comment by aspiwack** [comment]
Closed by #219
